plugins/module_utils/azure_rm_common.py

```diff
@@ -1187,6 +1229,11 @@ def postgresql_flexible_client(self):
                                                                         base_url=self._cloud_environment.endpoints.resource_manager)
         return self._postgresql_flexible_client
 
+    @property
+    def postgresql_flexible_models(self):
+        self.log("Getting postgresql_flexible models")
+        return PostgreSQLFlexibleModels
+
```

Comment:
I did this to follow what other code bas has done.  But I'm confused why we do this and try and import the model from the client module as well.  Is this legacy?  Which is the preferred way?

Reply:
I don't remember when this code was introduced, it existed before I joined the project. This is legacy code. It started as a simple way to manage models and clients and the client and models are mixed together.